packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1001,10 +1012,25 @@ export default class Meeting extends StatelessWebexPlugin {
         }
 
         this.requiredCaptcha = err.captchaInfo;
+
+        Metrics.sendBehavioralMetric(
```

Comment:
move this to meetingInfo v2 fetchMeetingInfo function

Reply:
Done, Moved all the sendBehavioralMetric from this to fetchMeetingInfo function